ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -197,4 +197,8 @@ default boolean addDynamicSplitPruningEdge(ExprNodeDesc syntheticFilterPredicate
   default Map<String, String> getOperatorDescProperties(OperatorDesc operatorDesc, Map<String, String> initialProps) {
     return initialProps;
   }
+
+  default Map<String, String> getBasicStatistics(TableDesc tableDesc) {
```

Comment:
Since it's a new interface method, can you add some javadoc please?

Reply:
Right, fixed it.